Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPITest.mm

```diff
@@ -218,6 +218,252 @@
     EXPECT_NS_EQUAL(firstMessage, @{ @"key": @"One" });
 }
 
+TEST(WKWebExtensionAPITest, AddAnonymousAsyncTest)
+{
+    auto *backgroundScript = Util::constructScript(@[
+        @"browser.test.assertRejects(browser.test.addTest(async () => {",
+        @"  browser.test.assertTrue(true)",
+        @"}))",
+        @"  .then(() => browser.test.notifyPass())",
+        @"  .catch(() => browser.test.notifyFail('Passing an anonymous function into addTest resolved the promise.'))"
+    ]);
+
+    auto manager = Util::loadExtension(manifest, @{ @"background.js": backgroundScript });
+
+    [manager run];
+    EXPECT_NS_EQUAL(manager.get().testsAdded, @[ ]);
```

Comment:
I'd add an empty line between the run and `EXPECT_NS_EQUAL` for all these new tests.
```suggestion
    [manager run];

    EXPECT_NS_EQUAL(manager.get().testsAdded, @[ ]);
```